content/tutorials/token-authentication.textile

```diff
@@ -320,7 +328,7 @@ blang[ruby].
 blang[python].
   "web.py":http://webpy.org/ is a simple web framework for Python. Django is arguably more popular as a web server framework, however for the purposes of this tutorial it is overkill.
 
-  Let's get going with a simple web.py web server now. Install @web.py@ using @pip@:
+  You'll get going with a simple web.py web server now. Install @web.py@ using @pip@:
```

Comment:
Probably retain 'Let's' in place of 'You'll' as the sentence loses context otherwise.